eventmesh-connector-plugin/eventmesh-connector-standalone/src/main/java/org/apache/eventmesh/connector/standalone/broker/task/SubScribeTask.java

```diff
@@ -57,8 +57,9 @@ public void run() {
                 if (offset == null) {
                     CloudEvent message = standaloneBroker.getMessage(topicName);
                     if (message != null) {
-                        if (message.getExtension("offset") != null) {
-                            offset = new AtomicInteger((int) message.getExtension("offset"));
+                        Object tmpOffset = message.getExtension("offset");
+                        if (tmpOffset instanceof Integer) {
+                            offset = new AtomicInteger((Integer) tmpOffset);
```

Comment:
Here please change to `new AtomicInteger(Integer.parseInt(tmpOffset.toString()))`